src/tracer.js

```diff
@@ -74,6 +74,7 @@ export default class Tracer {
             spanContext: SpanContext,
             operationName: string,
             startTime: number,
+            internalTags: any = [],
```

Comment:
I'm not worried about it, because a lot of people do this manually in es5 code.  Even in tchannel this is the pattern.